source/documentation/using_ci/index.md

```diff
@@ -0,0 +1,13 @@
+# Configuring a custom continuous integration (CI) system
+
+Setting up a CI service lets you make sure your code changes are delivering new features without introducing new bugs or breaking existing functionality. This helps you to iterate code faster and with greater confidence, a huge benefit once the codebase becomes larger and more complex. 
+ 
+For advanced projects that need high levels of control over configuration, a tool like [Jenkins](https://jenkins.io/) is often used. For simpler projects, or where a team needs to manage deployments of prototypes from their github repositories rather than local code, cloud-based services such as [Travis CI](https://travis-ci.com/) or [CircleCI](https://circleci.com/) are common. Instructions to configure both Jenkins and Travis to run on GOV.UK PaaS are below.
```

Comment:
I've seen that our trying to be abstract here can actually make people more confused (e.g. they don't know why we refer to 'Cloud Foundry' when they are working with PaaS). And not all our users are *strictly* 'devs'. I'm very happy to be opinionated if it makes things clearer for the most inexperienced users - remember, we have to make it so simple the majority of users succeed first time. But perhaps I'll add a "other CI services are also available" line afterwards.